src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/controllers/NotificationsController.java

```diff
@@ -336,4 +342,30 @@
 
         return ResponseEntity.ok(AboutToStartOrSubmitCallbackResponse.builder().data(updatedDetails.getData()).build());
     }
+
+    @PostMapping(value = "/case-orchestration/notify/manual-payment", consumes = APPLICATION_JSON_VALUE)
+    @ApiOperation(value = "send a manual payment letter")
+    @ApiResponses(value = {
+        @ApiResponse(code = 204, message = "Manual Payment letter sent successfully",
+            response = AboutToStartOrSubmitCallbackResponse.class)})
+    public ResponseEntity<AboutToStartOrSubmitCallbackResponse> sendManualPayment(
+        @RequestHeader(value = AUTHORIZATION_HEADER) String authToken,
+        @RequestBody CallbackRequest callbackRequest) {
+        log.info("Received request to send Manual Payment Letter for Case ID: {}", callbackRequest.getCaseDetails().getId());
+        validateCaseData(callbackRequest);
+
+        CaseDetails caseDetails = callbackRequest.getCaseDetails();
+
+        if (isContestedPaperApplication(caseDetails)) {
+            CaseDocument applicantManualPaymentLetter =
+                manualPaymentDocumentService.generateManualPaymentLetter(caseDetails, authToken, APPLICANT);
```

Comment:
you should have 2 seperate methods here, generateApplicantManualPaymentLetter and generateRespondentManualPaymentLetter, these methods should do whatever is specific to those and then call on to a generateManualPaymentLetter method where all of the shared code will live. See printApplicantDocuments in BulkPrintService as an example. You should never use strings to control logic, very bad practice